BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using BDArmory.Competition;
 using BDArmory.Control;
+using BDArmory.Modules;
```

Comment:
There's several unused `using` modules here that could be cleaned up.

Reply:
Thanks, IntelliNonsense.